giraffe/src/utils/rectTooltip.ts

```diff
@@ -39,11 +39,9 @@ export const findHoveredRects = (
   } else if (binDimension === 'x') {
     return xIndices
   }
-
```

Comment:
in general I'd avoid including these changes that don't do anything. it adds you as the last person who modified this file in git. Make sure you are using `git add --patch` to stage your changes one by one.  :)